src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java

```diff
@@ -30,35 +30,77 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
 
         String message;
         for (Mentor mentor : mentors) {
-            message = "You have been " + mentor.getState().name().toLowerCase();
-            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message);
+
+            if (mentor.getState().name().equals("APPROVED")) {
+
+                message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                        "<b>Congratulations!</b><br />You have been selected by the " +
+                        "ScholarX committee to be a mentor of the " + program.get().getTitle() +
+                        " program. We will soon open up the program for students to " +
+                        "apply and keep you posted on the progress via email. Until " +
+                        "then, read more about student experience " +
+                        "<a href=\"#\">here</a> and reach out to us via " +
+                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a> " +
+                        "for any clarifications.";
+
+                emailService.sendEmail(mentor.getProfile().getEmail(), "Approved", message, false);
+
+            } else if (mentor.getState().name().equals("REJECTED")) {
+
+                message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                        "Thank you very much for taking your time to apply for the " + program.get().getTitle() + " program. " +
+                        "However, due to the competitive nature of the mentor applications, your application " +
+                        "did not make it to the final list of mentors for the program. We encourage you to try " +
+                        "again next year and follow us on our social media channels for future programs. " +
+                        "If you have any clarifications, please reach out to us via " +
+                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>";
+
+                emailService.sendEmail(mentor.getProfile().getEmail(), "Rejected", message, false);
```

Comment:
Line 46 and 58. Let's use the program title here too without setting "rejected" or "approved" as the subject. 
I guess you can extract that line out of the if-else block call the sendEmail function only once as well that way.

Reply:
I didn't extract that line out of the if-else block because theoretically there could be other states except ACCEPTED and REJECTED and we don't send emails in those states

Reply:
Yeah makes sense.

Reply:
Done :) I changed it to use the program title